main/src/library/Eq.flix

```diff
@@ -47,7 +49,7 @@ pub lawful class Eq[a] {
     ///
     /// x != y is logically equivalent to not (x == y).
     ///
-    law inverseNeq: forall(x: a, y: a). (x != y) <==> (not (x == y))
```

Comment:
I realize I would also like to bring back `<==>` but keep it in the `Bool` namespace and import it.

Sorry for the confusion and thanks!